drivers/frequency/hmc7044/hmc7044.c

```diff
@@ -357,28 +357,36 @@ int32_t hmc7044_clk_round_rate(struct hmc7044_dev *dev, uint32_t rate,
 /**
  * Set channel rate.
  * @param dev - The device structure.
- * @param chan - Channel number.
+ * @param chan_num - Channel number.
  * @param rate - Channel rate.
  * @return SUCCESS in case of success, negative error code otherwise.
  */
-int32_t hmc7044_clk_set_rate(struct hmc7044_dev *dev, uint32_t chan,
+int32_t hmc7044_clk_set_rate(struct hmc7044_dev *dev, uint32_t chan_num,
```

Comment:
This seems like an unnecessary change

Reply:
This Commit should affect `hmc7044_clk_recalc_rate` in a similar fashion.

Reply:
Done.